x-pack/plugin/repository-encrypted/qa/azure/build.gradle

```diff
@@ -0,0 +1,8 @@
+
+apply plugin: 'elasticsearch.internal-cluster-test'
+apply plugin: 'elasticsearch.java'
+
+dependencies {
+    internalClusterTestImplementation testArtifact(project(':plugins:repository-azure'), 'internalClusterTest')
+    internalClusterTestImplementation testArtifact(project(':x-pack:plugin:repository-encrypted'), 'test')
```

Comment:
FWIW, in this scenario you can omit the `test` parameter as this is the default.

Reply:
I'll leave it for clarity, but feel free to remove it when I'm not looking